symphony-bdk-http/symphony-bdk-http-jersey2/src/main/java/com/symphony/bdk/http/jersey2/ApiClientBuilderJersey2.java

```diff
@@ -153,9 +177,20 @@ private ClientConfig createClientConfig() {
     // turn off compliance validation to be able to send payloads with DELETE calls
     clientConfig.property(ClientProperties.SUPPRESS_HTTP_COMPLIANCE_VALIDATION, true);
 
+    if (proxyUrl != null) {
+      configureProxy(clientConfig);
+    }
+
     return clientConfig;
   }
 
+  private void configureProxy(ClientConfig clientConfig) {
+    clientConfig.connectorProvider(new ApacheConnectorProvider());
+    clientConfig.property(ClientProperties.PROXY_URI, proxyUrl);
+    clientConfig.property(ClientProperties.PROXY_USERNAME, proxyUser);
```

Comment:
is that ok to set it to a null value?

Reply:
Yes, https://eclipse-ee4j.github.io/jersey.github.io/apidocs/latest/jersey/index.html?org/glassfish/jersey/client/ClientProperties.html says for PROXY_* fields that default value is null.